crates/compiler/builtins/roc/Dict.roc

```diff
@@ -1235,3 +1270,25 @@ expect
         |> complete
 
     hash1 != hash2
+
+expect
+    empty {}
+    |> len
+    |> Bool.isEq 0
+
+expect
+    empty {}
+    |> insert "One" "A Song"
+    |> insert "Two" "Candy Canes"
+    |> insert "Three" "Boughs of Holly"
+    |> clear
+    |> len
+    |> Bool.isEq 0
+
+expect
+    Dict.empty {}
+    |> Dict.insert "Apples" 12
+    |> Dict.insert "Orange" 24
+    |> Dict.insert "Banana" 36
+    |> Dict.walkUntil 0 (\count, k, qty -> if k == "Orange" then Break count else Continue (count + qty))
+    |> Bool.isEq 12
```

Comment:
These `expect`s don't compile for me. Did they work for you? I'm using `roc test crates/compiler/builtins/roc/Dict.roc` to test it. 

```
roc test crates/compiler/builtins/roc/Dict.roc 
An internal compiler expectation was broken.
This is definitely a compiler bug.
Please file an issue here: https://github.com/roc-lang/roc/issues/new/choose
thread '<unnamed>' panicked at 'Module 5 not found in world abilities', crates/compiler/late_solve/src/lib.rs:76:32
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
When you run anything in the builtins, make sure to do `cargo run test ...`. Due to builtin caching and link directly to the compiler, they sometimes break if you just run with a precompiled roc binary. Also, after rebasing on main, those tests run fine for me.

Reply:
Thank you @bhansconnect those tests pass for me too.